src/ui-lib/sass/05-components/03-organisms/_job-list.scss

```diff
@@ -1,12 +1,29 @@
 .grav-c-job-list {
-  > p:first-of-type {
+  margin-top: $grav-sp-xl;
+
+  @media all and (min-width: gravy-breakpoint(regular)) {
+    display: grid;
+    grid-template-columns: 30% 1fr;
+    grid-gap: $grav-sp-m;
+  }
+
+  > header > p {
     @include typi(small-text);
-    margin-top: $grav-sp-m;
+    margin-top: 0;
   }
 
   > ul {
     padding-left: $grav-sp-m;
```

Comment:
I think this should be `0` on mobile too?

Reply:
correct!!!!